client/commands/cutoff.js

```diff
@@ -51,6 +51,28 @@ const requestRate = () => {
   })
 }
 
+
+/**
+ * Calculates the standard error of 
+ * @param {Array} Array of all data points
+ * @param {Object} Linear Regression Model
+ * @param {Integer} Calculated Final Rate of the event
```

Comment:
stdError returns a result, so you'll want to add it into the comments
`@return {Integer} standard error`